app/controllers/orders/active_items.js

```diff
@@ -34,6 +82,11 @@ export default detail.extend({
         });
     },
 
+    applySortOn(options) {
+      this.set("setDefaultSortingColumn", options);
```

Comment:
notice the duplication of "set" ... that's why the name of this property is weird